run.py

```diff
@@ -131,6 +131,15 @@ def healthcheck_app():
     return client.response
 
 
+@app.route('/status', methods=["GET"])
+def status():
+    """
+    Uptime robotなどの稼働状況監視サービスが
+    アクセスするためのjsonを表示します
```

Comment:
```suggestion
    死活監視のためのレスポンスをJSON形式で返します
```

こちらの方が意味が分かりやすそうかなぁ。